package.json

```diff
@@ -99,7 +98,6 @@
   "workspaces": [
     "example"
   ],
-  "packageManager": "yarn@3.6.1",
```

Comment:
Hey, the yarn & yarn workspace is used for working on the sdk locally. I'll rename the example to avoid that confusion. RN devs should refer sample from [samples/example_new_arch](https://github.com/reclaimprotocol/reclaim-inapp-reactnative-sdk/tree/main/samples/example_new_arch)